apl/reference/special-field-attributes.mdx

```diff
@@ -44,13 +44,109 @@ Replace `FIELDNAME` with the actual name of the field.
+
+**Percentage**
+| Unit name | APL sytax |
+| --- | --- |
+| percent (0-100) | percent100 |
+| percent (0.0-1.0) | percent |
+
+**Currency**
+| Unit name | APL sytax |
+| --- | --- |
+| Dollars ($) | curusd |
+| Pounds (£) | curgbp |
+| Euro (€) | cureur |
+| Bitcoin (฿) | curbtc |
+
+**Data (IEC)**
+| Unit name | APL sytax |
+| --- | --- |
+| bits(IEC) | bits |
+| bytes(IEC) | bytes |
+| kibibytes | kbytes |
+| mebibytes | mbytes |
+| gibibytes | gbytes |
+| tebibytes | tbytes |
+| pebibytes | pbytes |
+
+**Data (metric)**
+| Unit name | APL sytax |
+| --- | --- |
+| bits(Metric) | decbits |
+| bytes(Metric) | decbytes |
+| kilobytes | deckbytes |
+| megabytes | decmbytes |
+| gigabytes | decgbytes |
+| terabytes | dectbytes |
+| petabytes | decpbytes |
+
+**Data rate**
+| Unit name | APL sytax |
+| --- | --- |
+| packets/sec | pps |
+| bits/sec | bps |
+| bytes/sec | Bps |
+| kilobytes/sec | KBs |
+| kilobits/sec | Kbits |
+| megabytes/sec | MBs |
+| megabits/sec | Mbits |
+| gigabytes/sec | GBs |
+| gigabits/sec | Gbits |
+| terabytes/sec | TBs |
+| terabits/sec | Tbits |
+| petabytes/sec | PBs |
+| petabits/sec | Pbits |
+
+**Datetime**
+| Unit name | APL sytax |
+| --- | --- |
+| Hertz (1/s) | hertz |
+| nanoseconds (ns) | ns |
+| microseconds (µs) | µs |
+| milliseconds (ms) | ms |
+| seconds (s) | secs |
+| minutes (m) | mins |
+| hours (h) | hours |
+| days (d) | days |
+| ago | ago |
+
+**Throughput**
+| Unit name | APL sytax |
+| --- | --- |
+| counts/sec (cps) | cps |
+| ops/sec (ops) | ops |
+| requests/sec (rps) | reqps |
+| reads/sec (rps) | rps |
+| writes/sec (wps) | wps |
+| I/O ops/sec (iops) | iops |
+| counts/min (cpm) | cpm |
+| ops/min (opm) | opm |
+| requests/min (rps) | reqpm |
+| reads/min (rpm) | rpm |
+| writes/min (wpm) | wpm |
+
 ## Example
 
-The example APL query below adds a tooltip and a description to the values of the `status` field. Clicking one of the values in this field leads to a page about status codes.
+The example APL query below adds a tooltip and a description to the values of the `status` field. Clicking one of the values in this field leads to a page about status codes. The unit of the request duration field is changed to nanoseconds.
 
 ```apl
-['http-logs']
-| extend _status_tooltip = "The status of the HTTP request is the response code from the server. It shows if an HTTP request has been successfully completed."
-| extend _status_description = "This is the status of the HTTP request."
-| extend _status_url = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status"
-```
\ No newline at end of file
+['sample-http-logs']
+| extend _status_tooltip = 'The status of the HTTP request is the response code from the server. It shows if an HTTP request has been successfully completed.'
+| extend _status_description = 'This is the status of the HTTP request.'
+| extend _status_url = 'https://developer.mozilla.org/en-US/docs/Web/HTTP/Status'
+| extend _req_duration_ms_unit = 'ns'
```

Comment:
i think this is the only thing i'd change so that it matches someone's real-world use case better. for example

```
| extend _resp_body_size_unit = "bytes"
```

Reply:
@kevinehosford Thanks! I agree the example is a bit contrived. I want to show that you can change the unit of the field with this attribute. Following your example, would this make more sense?

```
['sample-http-logs'] | extend _status_tooltip = 'The status of the HTTP request is the response code from the server. It shows if an HTTP request has been successfully completed.' | extend _status_description = 'This is the status of the HTTP request.' | extend _status_url = 'https://developer.mozilla.org/en-US/docs/Web/HTTP/Status' | extend _resp_body_size_bytes_unit = 'bits'
```

https://play.axiom.co/axiom-play-qf1k/explorer?qid=RGJZvYarhvR-snk35e&relative=1